libgstd/gstd_no_reader.c

```diff
@@ -74,8 +74,6 @@ static GstdReturnCode
 gstd_no_reader_read (GstdIReader * iface, GstdObject * object,
     const gchar * name, GstdObject ** out)
 {
-  GST_ERROR_OBJECT (iface, "Unable to read from this resource");
```

Comment:
Suppressing the error is the way to go here? Is it a critical error we want to take into account for debugging or can it be muted?

Reply:
We discussed it internally and agreed that the error handling should be the responsibility  of the caller and the no_reader should only return NULL.

In the only case where the no_reader is actually being called (trying to read from event objects) we shouldn't report an error because the events don't create an object.